Source/Core/DolphinQt/GameList/GameFile.cpp

```diff
@@ -1,283 +1,78 @@
-// Copyright 2014 Dolphin Emulator Project
+// Copyright 2015 Dolphin Emulator Project
 // Licensed under GPLv2+
 // Refer to the license.txt file included.
 
-#include <algorithm>
-#include <memory>
-
+#include <QCryptographicHash>
 #include <QDataStream>
 #include <QDir>
 #include <QFile>
 #include <QFileInfo>
-#include <QXmlStreamReader>
+#include <QImage>
+#include <QSharedPointer>
 
-#include "Common/Common.h"
-#include "Common/CommonPaths.h"
 #include "Common/FileUtil.h"
-#include "Common/Hash.h"
-#include "Common/IniFile.h"
-#include "Common/StringUtil.h"
-
 #include "Core/ConfigManager.h"
-
-#include "DiscIO/CompressedBlob.h"
-#include "DiscIO/Filesystem.h"
-
+#include "DiscIO/VolumeCreator.h"
+#include "DolphinQt/Resources.h"
 #include "DolphinQt/GameList/GameFile.h"
-#include "DolphinQt/Utils/Utils.h"
 
-static const u32 CACHE_REVISION = 0x00D; // Last changed in PR 3097
```

Comment:
I'd like to keep the "Last changed in..." comment. It's useful for detecting if a PR tries to increment the version number at the same time as another PR.